redirects/16.0.txt

```diff
@@ -58,13 +58,13 @@ applications/inventory_and_mrp/inventory/management/products/mto.rst application
 applications/inventory_and_mrp/inventory/management/products/reordering_rules.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst                                      # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
 applications/inventory_and_mrp/inventory/management/products/uom.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst                                                                # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
 applications/inventory_and_mrp/inventory/management/products/scrap_inventory.rst applications/inventory_and_mrp/inventory/product_management/product_replenishment/scrap_inventory.rst                                        # /inventory_and_mrp/inventory/management/products/* -> /inventory_and_mrp/inventory/product_management/product_replenishment/*
+applications/inventory_and_mrp/inventory/product_management/product_replenishment/scrap_inventory.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory.rst                    # /inventory_and_mrp/inventory/product_management/product_replenishment/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
```

Comment:
Small update to the comment– since only a file is being moved an not a whole directory
```suggestion
applications/inventory_and_mrp/inventory/product_management/product_replenishment/scrap_inventory.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory.rst                    # /inventory_and_mrp/inventory/product_management/product_replenishment/scrap_inventory -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory
```